fe/fe-core/src/main/java/org/apache/doris/datasource/InternalDataSource.java

```diff
@@ -1718,6 +1713,26 @@ private void createOlapTable(Database db, CreateTableStmt stmt) throws UserExcep
         // this should be done before create partition.
         Map<String, String> properties = stmt.getProperties();
 
+        // get use light schema change
+        Boolean useLightSchemaChange = true;
+        try {
+            useLightSchemaChange = PropertyAnalyzer.analyzeUseLightSchemaChange(properties);
```

Comment:
Does useLightSchemaChange only control whether a column has unique id? What happens when alter table?

Reply:
Yes, this new property only affects whether the column would be assigned a unique id or not.
Alter table job will try to use the light schema change optimization (like add new value column) if the columns of the tables are assigned unique id.